src/twisted/conch/checkers.py

```diff
@@ -13,14 +13,18 @@
 from base64 import decodebytes
 
 try:
-    import pwd
+    import pwd as _pwd
 except ImportError:
-    pwd = None  # type: ignore[assignment]
+    pwd = None
+else:
+    pwd = _pwd
```

Comment:
Using @cjerdonek's [workaround](https://github.com/python/mypy/issues/1297#issuecomment-508593494) to "Incompatible types in assignment" errors throughout this PR.

I found that `# type: ignore` as a work-around brings along other fallout, including that it confuses Mypy when checking usage of these symbols later in the file.

Reply:
Ah ok, in my first go around with mypy, I chose to go with adding `#type: ignore[assignment]` instead of going with the [workaround](https://github.com/python/mypy/issues/1297#issuecomment-508593494).

Your change looks fine.

In future, I hope that mypy can handle this kind of conditional import, but I don't think that will happen soon.